business-materials/01_事業内容一覧_社会的意義.md

```
@@ -2,43 +2,122 @@
+> **競争ではなく共創を通じて未来を切り拓き、幸福な社会を実現する。**
+
+### Vision
+
+> **多様な認知特性や価値観を持つ全ての人が、誤解なく互いを理解し合える世界を創る**
 
 ### Value
+
 1. テクノロジーと人間性のバランスを常に意識する
 2. 「当たり前」を疑い、異なる視点から解決策を模索する
-3. スピード感を活かし、創造性と革新性に満ちたサービスを提供する
+3. スピード感を活かし、創造性と革新性に満ちたサービスを提供する
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)